utils/_context/containers.py

```diff
@@ -181,12 +187,20 @@ def start(self, network: Network) -> Container:
         if self.healthy:
             self.warmup()
 
+        self._container.reload()
```

Comment:
why reload?

Reply:
To be sure to have fresh data, see my comment above